app/controllers/auth.controller.js

```diff
@@ -0,0 +1,60 @@
+const User = require("../models/user.model");
+const { validationResult } = require("express-validator");
+const bcrypt = require("bcrypt");
+
+/* Existing user Login Controller */
+exports.login = async (req, res) => {
+  const errors = validationResult(req);
+
+  if (!errors.isEmpty()) {
+    return res
+      .status(400)
+      .json({ status: false, full_messages: errors.array() });
+  }
+
+  const { email, password } = req.body;
+
+  try {
+    const userLogin = await User.findOne({ email });
+
+    if (!userLogin) {
+      return res
+        .status(400)
```

Comment:
I don't think 400 is appropriate here. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400

Maybe 401?